webdriver/README.md

```diff
@@ -1,8 +1,132 @@
 # WebDriver specification tests
 
-Herein lies a set of conformance tests
-for the W3C web browser automation specification
-known as [WebDriver](http://w3c.github.io/webdriver/webdriver-spec.html).
-The purpose of these tests is determine implementation compliance
-so that different driver implementations can determine
-whether they meet the recognised standard.
+Herein lies a set of conformance tests 
+for the W3C web browser automation specification 
+known as [WebDriver](http://w3c.github.io/webdriver/webdriver-spec.html). 
+The purpose of these tests is determine implementation compliance 
+so that different driver implementations can determine 
+whether they meet the recognised standard. 
+
+## How to run the tests
```

Comment:
I didn't write that part, it was already there. Do you want me to remove it for this PR?